package.json

```diff
@@ -120,10 +102,545 @@
     "ts-jest": "^28.0.3",
     "ts-loader": "^9.3.0",
     "typescript": "^4.7.3",
+    "victory-vendor": "*",
+    "victory-voronoi": "*",
     "webpack": "^5.72.1",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.9.0",
-    "webpack-stats-plugin": "^1.0.3"
+    "webpack-stats-plugin": "^1.0.3",
+    "wireit": "^0.7.1"
+  },
+  "pnpm": {
+    "overrides": {
+      "@types/eslint": "8.4.3"
+    }
+  },
+  "scripts": {
+    "start": "concurrently \"pnpm:build:lib:esm --watch\" \"webpack serve --config ./config/webpack/demo/webpack.config.dev.js --static demo/js --entry ./demo/js/app\"",
+    "start:ts": "concurrently \"pnpm:build:typescript --watch\" \"webpack serve --config ./config/webpack/demo/webpack.config.dev.js --static demo/ts --entry ./demo/ts/app\"",
+    "check": "wireit",
+    "check:ci": "wireit",
+    "check:debug": "cross-env WIREIT_PARALLEL=1 pnpm check",
+    "clean:build": "rimraf coverage \"packages/*/{dist,es,lib}\"",
+    "clean:cache": "wireit",
+    "clean:cache:lint": "rimraf .eslintcache \"packages/*/.eslintcache\"",
+    "clean:cache:wireit": "rimraf .wireit \"packages/*/.wireit\"",
+    "clean:cache:modules": "rimraf node_modules/.cache \"packages/*/node_modules/.cache\"",
+    "build": "wireit",
+    "build:lib:esm": "wireit",
+    "build:typescript": "wireit",
+    "format": "wireit",
+    "format:fix": "wireit",
+    "format:root": "wireit",
+    "format:root:fix": "wireit",
+    "format:pkgs": "wireit",
+    "format:pkgs:fix": "wireit",
+    "lint": "wireit",
+    "lint:fix": "wireit",
+    "lint:root": "wireit",
+    "lint:root:fix": "wireit",
+    "lint:pkgs": "wireit",
+    "lint:pkgs:fix": "wireit",
+    "lint:docs": "echo TODO FIGURE OUT INTEGRATION",
+    "jest": "wireit",
+    "jest:root": "wireit",
+    "jest:pkgs": "wireit",
+    "types:check": "wireit",
+    "types:create": "wireit",
+    "storybook:build": "wireit",
+    "storybook:server": "concurrently \"pnpm:build:lib:esm --watch\" \"start-storybook -p 6006\"",
```

Comment:
This script could (eventually) use some improvement.  `concurrently` breaks Storybook's/webpack's logging, so every tiny change results in 500 log messages (`"[webpack.Progress]"`).  Just a minor nuisance.

Reply:
Can you try again? I've added the `--raw` flag to all our concurrently use which should make this and the `start*` tasks work better in a terminal and not blast output.

Reply:
Way better!  Messages are way more visible now, and errors don't get lost!